app/views/collaborators/_collaborator.html.erb

```diff
@@ -1,20 +1,19 @@
- 
+<h4>Add Collaborators </h4>
+<table style="width:50%">
  <div class="form-group">
     <%# this should more properly use a variable set in your controller rather than User.all - for example you might want to limit the list of possible collaborators according to some condition %>
     <% User.all.each do |user| %>
-      <% #if collaborator = user.collaborator_for(wiki) %>
-      
-      <% if wiki.private? && user != wiki.user && !(collaborator = user.collaborator_for(wiki, user)) %>
-        <div class="user">
-          <%= user.name %> 
-          <%= link_to  "make collaborator", collaborators_path(wiki_id: wiki.id, user_id:user.id), class: 'btn btn-primary', method: :post %>
-        </div>
-      <% end %>
-      <% if wiki.private? && user != wiki.user && (collaborator = user.collaborator_for(wiki,user)) %>
-        <%#= raise collaborator.inspect %>
-        <%= user.name %> 
-        <%= link_to "Remove_collaborator", collaborator_path(user.collaborator_for(wiki,user)), class: 'btn btn-danger', method: :delete %>
-        <%#= link_to "delete collaborator", [wiki, collaborator], method: :delete do %>
-      <% end %>
+      <tr>
+        <% if user != wiki.user %>
+          <td><%= user.name %></td>
+        <% end %>
+        <% if wiki.private? && user != wiki.user && !(collaborator = user.collaborator_for(wiki, user)) %>
```

Comment:
Review `collaborator = user.collaborator_for ...` on lines 10 and 13. Why is their a `collaborator` local variable being assigned?